### ISSUE TITLE
[WARNING] Duplicates found in metadata. Keeping only first instance of each duplicated value

### ISSUE THREAD

**Issue by MrDibbley** [Living-with-machines/MapReader#398]
**Describe the bug**
If metadata contains an image that covers two sheets ie:

```json
 {
            "type": "Feature",
            "id": "6in_Eng_Scot_Wal_1st.5547",
            "geometry": {
                "type": "MultiPolygon",
                "coordinates": [
                    [
                        [
                            [
                                -4.5665,
                                50.8858
                            ],
                            [
                                -4.5664,
                                50.9147
                            ],
                            [
                                -4.4977,
                                50.9145
                            ],
                            [
                                -4.4979,
                                50.8856
                            ],
                            [
                                -4.5665,
                                50.8858
                            ]
                        ]
                    ]
                ]
            },
            "geometry_name": "the_geom",
            "properties": {
                "IMAGE": "101724551",
                "SERIES": "OS 6 Inch to the Mile - England & Wales",
                "SET": "6Q_A",
                "COUNTRY": "ENG",
                "COUNTY": "Cornwall",
                "SHEET_MAP": "I.SW & SE",
                "SHEET_NO": "001_SE",
                "EDITION": "1",
                "INSET_EXT": "",
                "RESURVEY": "",
                "SUR_STA": "1884",
                "SUR_END": "1884",
                "PSUR_STA": "",
                "PSUR_END": "",
                "ENG_STA": "",
                "ENG_END": "",
                "REV_STA": "",
                "REV_END": "",
                "PUB_STA": "1884",
                "PUB_END": "1884",
                "PPUB_STA": "",
                "PPUB_END": "",
                "PRI_STA": "",
                "PRI_END": "",
                "RPRI_STA": "",
                "RPRI_END": "",
                "MSD_SIG": "",
                "LEV_STA": "",
                "LEV_END": "",
                "BOUND_STA": "",
                "BOUND_END": "",
                "SUR_SORT": "1884",
                "PUB_SORT": "1884",
                "PARISH": "Morwenstow",
                "RAILWAYS": "",
                "RECORDSET": "",
                "BNDRY_INST": "",
                "YEAR": "1884",
                "GROUP": "102",
                "IN_NLS": "Y",
                "NOTE_MAP": "",
                "NOTE_LIST": "",
                "SHEET": "Cornwall I.SW & SE",
                "DATES": "Series: Ordnance Survey. Six-inch to the mile<br>Surveyed: 1884<br>Published: 1884",
                "IMAGETHUMB": "https://deriv.nls.uk/dcn4/1017/2455/101724551.4.jpg",
                "IMAGEURL": "https://maps.nls.uk/view/101724551",
                "IIIF_JSON": "https://map-view.nls.uk/iiif/2/10172%2F101724551/info.json",
                "WFS_TITLE": "Cornwall I.SW & SE, Surveyed: 1884, Published: 1884",
                "WFS": "Y",
                "OR_Num": "0"
            },
            "bbox": [
                -4.5665,
                50.8856,
                -4.4977,
                50.9147
            ]
        },
        {
            "type": "Feature",
            "id": "6in_Eng_Scot_Wal_1st.5549",
            "geometry": {
                "type": "MultiPolygon",
                "coordinates": [
                    [
                        [
                            [
                                -4.6351,
                                50.8859
                            ],
                            [
                                -4.635,
                                50.9148
                            ],
                            [
                                -4.5664,
                                50.9147
                            ],
                            [
                                -4.5665,
                                50.8858
                            ],
                            [
                                -4.6351,
                                50.8859
                            ]
                        ]
                    ]
                ]
            },
            "geometry_name": "the_geom",
            "properties": {
                "IMAGE": "101724551",
                "SERIES": "OS 6 Inch to the Mile - England & Wales",
                "SET": "6Q_A",
                "COUNTRY": "ENG",
                "COUNTY": "Cornwall",
                "SHEET_MAP": "I.SW & SE",
                "SHEET_NO": "001_SW",
                "EDITION": "1",
                "INSET_EXT": "",
                "RESURVEY": "",
                "SUR_STA": "1884",
                "SUR_END": "1884",
                "PSUR_STA": "",
                "PSUR_END": "",
                "ENG_STA": "",
                "ENG_END": "",
                "REV_STA": "",
                "REV_END": "",
                "PUB_STA": "1884",
                "PUB_END": "1884",
                "PPUB_STA": "",
                "PPUB_END": "",
                "PRI_STA": "",
                "PRI_END": "",
                "RPRI_STA": "",
                "RPRI_END": "",
                "MSD_SIG": "",
                "LEV_STA": "",
                "LEV_END": "",
                "BOUND_STA": "",
                "BOUND_END": "",
                "SUR_SORT": "1884",
                "PUB_SORT": "1884",
                "PARISH": "Morwenstow",
                "RAILWAYS": "",
                "RECORDSET": "",
                "BNDRY_INST": "",
                "YEAR": "1884",
                "GROUP": "102",
                "IN_NLS": "Y",
                "NOTE_MAP": "",
                "NOTE_LIST": "",
                "SHEET": "Cornwall I.SW & SE",
                "DATES": "Series: Ordnance Survey. Six-inch to the mile<br>Surveyed: 1884<br>Published: 1884",
                "IMAGETHUMB": "https://deriv.nls.uk/dcn4/1017/2455/101724551.4.jpg",
                "IMAGEURL": "https://maps.nls.uk/view/101724551",
                "IIIF_JSON": "https://map-view.nls.uk/iiif/2/10172%2F101724551/info.json",
                "WFS_TITLE": "Cornwall I.SW & SE, Surveyed: 1884, Published: 1884",
                "WFS": "Y",
                "OR_Num": "0"
            },
            "bbox": [
                -4.6351,
                50.8858,
                -4.5664,
                50.9148
            ]
        },
```
add metadata results in duplicates found (due to the same image being used for both sheets but each entry having different geo

see (https://github.com/rwood-97/tilelayer_finder/issues/1)

**Comment by rwood-97** [comment]
Just to let you know I probably won't have time to fix this until next week but will move it to the top of my priorities list for then.

**Comment by rwood-97** [comment]
Think I've fixed this in #413.
You can get these by installing mapreader==1.3.1

To get it working, you will need to delete the problem maps (i.e. both maps that link to that same NLS image URL "https://maps.nls.uk/view/101724551"). Either just delete your whole `./maps` directory, including the `metadata.csv`, and redownload them all or and just delete the problem png files then also delete these from your metadata.csv file.

Then you can redownload and it should create one map called `map_101724551.png` and then a second one called `map_101724551_1.png`. These will have separate coordinates in your metadata.

The only bit I **can't** fix is if you set `overwrite=True` when you are downloading for the first time (i.e. `map_101724551_1.png` is not yet in the metadata file, then when it comes to downloading the second map for "https://maps.nls.uk/view/101724551" it will end up overwriting the first and you'll see the same issue. 
So you will need to make sure `overwrite=False` when you are downloading.
